.github/workflows/pr.yaml

```diff
@@ -11,6 +11,8 @@ jobs:
       uses: ./
       with:
         channel: 1.24/edge
```

Comment:
While we're here, shouldn't we bump this to 1.25? Or use latest/stable?

Reply:
We can, let me add a commit for that